Source/CombatExtended/CombatExtended/CollisionVertical.cs

```diff
@@ -69,10 +70,12 @@ private static void CalculateHeightRange(Thing thing, out FloatRange heightRange
             
             float collisionHeight = 0f;
             float shotHeightOffset = 0;
+	    float heightAdjust = CETrenches.GetHeightAdjust(thing.Position, thing.Map);
```

Comment:
nitpick: inconsistent indent

Reply:
Yeah, I think a format tool will be high on the list of things to do before 4.5...